app/views/about/index.html.erb

```diff
@@ -124,7 +124,8 @@
       <h4><%= _("Support") %></h4>
       <h6><%= _("Documentation") %></h6>
       <ul>
-        <li><%= link_to _('Manual'),documentation_url, :rel => "external" %></li>
+        <li><%= link_to _('API'), apipie_apipie_path %></li>
```

Comment:
Yeah... I was not sure about the label, "API documentation" seems too much for just a list of endpoints. How about "Available API calls" or "API provided by the server"?